src/test/scala/mesosphere/marathon/api/v2/ModelValidationTest.scala

```diff
@@ -74,6 +80,34 @@ class ModelValidationTest
       v.message == "Requested service port 3200 conflicts with a service port in app /app2") should be(true)
   }
 
+  test("Multiple errors within one field of a validator should be grouped into one array") {
+    val empty = ImportantTitle("")
+
+    validate(empty) match {
+      case Success => fail()
+      case f: Failure =>
+        val errors = (Json.toJson(f) \ "details").as[Seq[JsObject]]
+        errors should have size 1
+        (errors.head \ "path").as[String] should be("name")
```

Comment:
As @alexanderweber  wrote
> So self would appear as a standalone information, if the error is not in an attribute, but the object itself.

That's how I got it too. If we have precise information regarding which attribute failed validation, then we don't need `self`. What's described at point 6 makes absolute sense to me:

> "path": "self" if there is no description about the point of failure at all